tests/conditional_test.py

```diff
@@ -36,9 +36,9 @@ def test_pytket_qir_conditional() -> None:
     c = circ.add_c_register("c", 5)
     d = circ.add_c_register("d", 5)
     circ.H(0)
-    circ.add_classicalexpbox_register(a | b, c)
-    circ.add_classicalexpbox_register(c | b, d)
-    circ.add_classicalexpbox_register(c | b, d, condition=a[4])
+    circ.add_classicalexpbox_register(a | b, c)  # type: ignore
```

Comment:
Mypy does not like this anymore, it only wants list of bits here, but I would like to keep it like this (same below)